.fixtures.yml

```diff
@@ -0,0 +1,17 @@
+# This file can be used to install module dependencies for unit testing
+# See https://github.com/puppetlabs/puppetlabs_spec_helper#using-fixtures for details
+
+---
+fixtures:
+  repositories:
+    stdlib:
+      repo: git://github.com/puppetlabs/puppetlabs-stdlib.git
+      ref: v6.0.0
+    archive:
+      repo: https://github.com/voxpupuli/puppet-archive.git
+      ref: v1.0.0
```

Comment:
archive is at 4.5.0

Reply:
This file is the lower bound tests, do we want to increase support archive version to require at minimum 4.5.0? The `.fixtures-latest.yml` tests master.

Reply:
This is fixed.